cmake/QSkinnyConfig.cmake

```diff
@@ -1 +1,3 @@
 include("${CMAKE_CURRENT_LIST_DIR}/QSkinnyTargets.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/QSkinnyTools.cmake" OPTIONAL)
+include("${CMAKE_CURRENT_LIST_DIR}/QSkinnyQmlExportTargets.cmake" OPTIONAL)
```

Comment:
include all possible CMake scripts and mark them optional if e.g. controlled by a CMake option